src/components/toast/index.js

```diff
@@ -252,5 +254,12 @@ function createStyles() {
       height: 12,
       tintColor: Colors.white,
     },
+    blur: {
```

Comment:
quicker/cleaner to use `StyleSheet.absoluteFillObject`
https://facebook.github.io/react-native/docs/stylesheet.html#absolutefillobject